docs/sources/setup-grafana/configure-security/configure-authentication/keycloak/index.md

```diff
@@ -109,7 +109,14 @@ This will add the `groups` claim to the id_token. You can then use the `groups`
 
 ```ini
 [auth.generic_oauth]
-group_attribute_path = groups
+groups_attribute_path = groups
+```
+
+If using nested groups that contain special characters such as quotes or colons, the JMESPath parser can perform a harmless reverse function so Grafana will properly evaluate nested groups. eg. a parent group named `Global` with nested group `department` containing a list of groups:
```

Comment:
```suggestion
If you use nested groups containing special characters such as quotes or colons, the JMESPath parser can perform a harmless reverse function so Grafana can properly evaluate nested groups. For example, a parent group named `Global` with nested group `department` containing a list of groups:
```

Reply:
@IevaVasiljeva , can you please take a look at this PR? If not you, who can help? Thank you!